feat: Add CloudQuery Cloud integration (ECOINT-35) (#2493)
DataDog/integrations-extras: cloudquery-cloud/README.md

# CloudQuery Cloud

## Overview

[CloudQuery][1] is an open-source, high-performance data integration framework built for developers, with support for a wide range of plugins.

CloudQuery extracts, transforms, and loads configuration from cloud APIs to a variety of supported destinations such as databases, data lakes, or streaming platforms for further analysis.

[CloudQuery Cloud][2] is a great way to get started with CloudQuery and syncing data from source to destination without the need to deploy your own infrastructure. It is also much easier to connect to sources and destinations with the integrated OAuth authentication support. You only need to select a source and destination plugin and CloudQuery will take care of the rest.

## Setup

### Installation

> Sign up for free at [cloud.cloudquery.io][2]. Once logged in, navigate to the **Sources** page and add a Datadog source. Under the **Authentication** section, use the **Authenticate** button to grant access to your Datadog account using OAuth2 flow.

Reviewer: ```suggestion
1. Sign up for free at [cloud.cloudquery.io][2]. 
2. In Datadog, navigate to the CloudQuery Cloud integration tile
3. Click **Connect Accounts**
4. You'll be redirected to CloudQuery to log in
5. Navigate to the **Sources** page and add a Datadog source
6. Under the **Authentication** section, use the **Authenticate** button to grant access to your Datadog account using OAuth2 flow.
```

# CloudQuery Cloud

> ![datadog-integration][1]

Reviewer: ```suggestion
```
We display a media carousel at the top of integration tiles. This is not needed.

## Overview

> [CloudQuery][2] is an open-source, high-performance data integration framework built for developers, with support for a wide range of plugins.

Reviewer: Within the overview section, we typically ask Partners to follow the below format: 
1. 1-2 sentences about what your platform does
2. 1-2 sentences about what this integration does and what kind of data it pulls/submits
3. 1-2 sentences about how this integration benefits customers

Could you be more specific about what CloudQuery does in your first sentence?

CloudQuery extracts, transforms, and loads configuration from cloud APIs to a variety of supported destinations such as databases, data lakes, or streaming platforms for further analysis.

[CloudQuery Cloud][3] is a great way to get started with CloudQuery and syncing data from source to destination without the need to deploy your own infrastructure. It is also much easier to connect to sources and destinations with the integrated OAuth authentication support. You only need to select a source and destination plugin and CloudQuery will take care of the rest.

## Setup

### Installation

> Sign up for free at [cloud.cloudquery.io][3]. Once logged in, navigate to the **Sources** page and add a Datadog source. Under the **Authentication** section, use the **Authenticate** button to grant access to your Datadog account using OAuth2 flow.

Reviewer: Could you reformat this to describe the instructions a user would take when starting the OAuth flow from Datadog? See an example [here](https://app.datadoghq.com/integrations/ably) on the configure tab.

For more information about using CloudQuery Cloud, refer to the [quickstart guide][4].

### Configuration

Detailed documentation for the CloudQuery Datadog source is available [here][5].

## Uninstallation

> Navigate to the **Sources** page under [CloudQuery Cloud][3] and find your Datadog source you have previously set up. Under the **Edit source** tab, click the **Delete this source** button.

Reviewer: ```suggestion
1. Navigate to the **Sources** page under [CloudQuery Cloud][3] and find your Datadog source you have previously set up. 
2. Under the **Edit source** tab, click the **Delete this source** button.
```

## Support

For support, contact [CloudQuery][2] or [CloudQuery Community][6].

> [1]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/cloudquery/images/cloudquery_logo_png_dark_background.png

Reviewer: Please reformat these footer links now that 1 no longer applies.
